### PR TITLE
allow query by multiple invoice ids

### PR DESCRIPTION
**Pull request by peillis**

### CODE REVIEW

File: lib/siwapp_web/graphql/schema.ex [hunk 1]
```diff
@@ -32,6 +32,7 @@ defmodule SiwappWeb.GraphQL.Schema do
 
     @desc "Get all invoices"
     field :invoices, list_of(:invoice) do
+      arg(:ids, list_of(non_null(:id)))
```
Comment (agutierrezrodriguez) [inline lib/siwapp_web/graphql/schema.ex:35]: **Medium — reject or normalize a null `ids` argument**

This optional list accepts `invoices(ids: null)`. Absinthe then supplies `ids: nil`; the oversize guard does not match and the generic resolver forwards it to `invoice.id in ^value`, where Ecto expects a list and raises a cast error. A valid GraphQL request can therefore become a server error. Make the argument non-null when supplied (for example, `non_null(list_of(non_null(:id)))`) or drop/normalize `ids: nil` before building filters, and add a regression test.

Source confidence: raised by one reviewer; independently verified against the schema, resolver, and query path.

File: lib/siwapp_web/graphql/schema.ex [hunk 2]
```diff
@@ -32,6 +32,7 @@ defmodule SiwappWeb.GraphQL.Schema do
...
       arg(:customer_id, :id)
       arg(:limit, :integer, default_value: 10)
```
Comment (agutierrezrodriguez) [inline lib/siwapp_web/graphql/schema.ex]: **Medium — do not silently truncate an ID batch at the ordinary page size**

A caller using the new `ids` argument without also knowing to override `limit` receives only the first 10 matches because this existing default is still applied. The new happy-path test hides the behavior by explicitly setting `limit: 100`; it contradicts the advertised ability to query a batch of up to 100 invoice IDs. When `ids` is present, set the effective limit from the batch (or otherwise reject/document pagination explicitly) and add a test that omits `limit`.

Source confidence: raised by two reviewers; independently verified against the resolver and `Invoices.list/1`.

**Review by agutierrezrodriguez — CHANGES_REQUESTED** [review]
## Changes requested

Two GraphQL edge cases in the new `ids` filter need to be resolved before merge. CI is green; the issues are not covered by the added tests.

File: lib/siwapp_web/graphql/schema.ex (hunk 1 above)
Comment (peillis) [reply lib/siwapp_web/graphql/schema.ex:35]: fixed

File: lib/siwapp_web/graphql/schema.ex (hunk 2 above)
Comment (peillis) [reply lib/siwapp_web/graphql/schema.ex]: fixed

**Review by agutierrezrodriguez — APPROVED** [review]
## Approval

No material findings in the accessible PR scope. The current `test-elixir (1.19.4, 28.1.1)` check passed.

## Human review decision
Human review required: no
The full accessible change set and relevant automated check were reviewed with no unresolved material concerns.
